docs/scripts_md/MRI.md

```diff
@@ -292,8 +292,9 @@ object referenced by `$file_ref`.
 
 INPUTS:
   - $file\_ref      : file hash ref
-  - $data\_dir      : data directory (C/data/$PROJECT/data> typically)
-  - $dest\_dir      : destination directory (`/data/$PROJECT/data` typically)
+  - $data\_dir      : data directory (`/data/$PROJECT/data` typically)
+  - $dest\_dir      : destination directory (`/data/$PROJECT/data/pics`
```

Comment:
you will kill me for this! I just double checked; it is `pic` not `pics`
I will merge right after this small change!

Reply:
@MounaSafiHarab which poison do you prefer? ;) Very nice catch. Your new name is going to be "Oeil de Lynx"! Anyway, I just fixed it